Tools/Scripts/compare-results

```diff
@@ -466,14 +512,20 @@ def detectMotionMark1_1(payload):
 def detectMotionMark1_1_1(payload):
     return "MotionMark-1.1.1" in payload
 
+def detectMotionMark1_2(payload):
+    return "MotionMark-1.2" in payload
+
+
 def motionMarkResults(payload):
-    assert detectMotionMark(payload) or detectMotionMark1_1(payload) or detectMotionMark1_1_1(payload)
+    assert detectMotionMark(payload) or detectMotionMark1_1(payload) or detectMotionMark1_1_1(payload) or detectMotionMark1_2(payload)
```

Comment:
Nit: This can be rewritten as:
```
assert any(validMotionMarkDetector(payload) for validMotionMarkDetector in [detectMotionMark, detectMotionMark1_1, detectMotionMark1_1_1, detectMotionMark1_2])
```